### PR TITLE
Switch from CircleCI back to Travis

### PR DESCRIPTION
**Pull request by mihails-strasuns**

### CODE REVIEW

**Comment by mihails-strasuns** [comment]
@leandro-lucarella-sociomantic you can disable circleci when merging this - I don't have org level admin access to do so myself.

**Comment by leandro-lucarella-sociomantic** [comment]
LGTM, can you force push to verify circleci is disabled?

File: docker/build [hunk 1]
```diff
@@ -1,22 +1,9 @@
...
-if [ "`echo $DMD_PKG | grep dmd-compiler`" ]
-then
-    # Remove pre-installed DMD package to avoid conflicts
-    apt-get remove -y dmd-bin
-fi
-
 # Install dependencies
 apt-get install -y \
-    $DMD_PKG \
+    d1to2fix=0.10.* \
```
Comment (leandro-lucarella-sociomantic) [inline docker/build:6]: Why is this necessary?

**Review by leandro-lucarella-sociomantic — APPROVED** [review]

File: docker/build (hunk 1 above)
Comment (mihails-strasuns) [reply docker/build:6]: I copied current ocean build script blindly, nothing is intentional :)

**Comment by mihails-strasuns** [comment]
Force pushed

File: docker/build (hunk 1 above)
Comment (leandro-lucarella-sociomantic) [reply docker/build:6]: ping
